functions/helpers.bash

```diff
@@ -177,7 +177,7 @@ is_raspbian() {
   [[ "$(cat /etc/*release*)" =~ "Raspbian" ]]
   return $?
 }
-# Debian/Raspbian, to be deprecated, LTS ends 2020-06-30
+# Debian/Raspbian, deprecated, LTS ended 2020-06-30
```

Comment:
As an aside, my computer claims that the date is only 2020-06-16 which means that this claim is currently false.

Reply:
hehehe I was wondering if anyone would notice :)
But I didn't want to do another PR in 2 weeks